index.ts

```diff
@@ -68,14 +69,30 @@ export class HttpClientResponse implements ifm.IHttpClientResponse {
     public message: http.IncomingMessage;
```

Comment:
I'm still trying to figure out why `message` can't reliably be used to read a gzipped response body. Maybe because message.on('Data') is returning strings - with the wrong encoding - instead of buffers?

Reply:
Looks like `message` can be used. [Slack discussion](https://github.slack.com/archives/CR1FUMRPX/p1585772094017800?thread_ts=1585676936.005700&cid=CR1FUMRPX)